src/parser/warlock/affliction/modules/talents/VileTaint.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import { formatThousands } from 'common/format';
+
+import Analyzer from 'parser/core/Analyzer';
+import AbilityTracker from 'parser/shared/modules/AbilityTracker';
+
+import SpellLink from 'common/SpellLink';
+import StatisticListBoxItem from 'interface/others/StatisticListBoxItem';
+
+class VileTaint extends Analyzer {
+  static dependencies = {
+    abilityTracker: AbilityTracker,
+  };
+
+  constructor(...args) {
+    super(...args);
+    this.active = this.selectedCombatant.hasTalent(SPELLS.VILE_TAINT_TALENT.id);
+  }
+
+  subStatistic() {
+    const spell = this.abilityTracker.getAbility(SPELLS.VILE_TAINT_TALENT.id);
+    const damage = spell.damageEffective + spell.damageAbsorbed;
+    return (
+      <StatisticListBoxItem
```

Comment:
Would it be beneficial to display how many targets are hit on average? Maybe as an addition to the tooltip.